.multipackage/scripts/generate_api.py

```diff
@@ -0,0 +1,140 @@
+"""Generate API docs for all subpackages."""
+
+from __future__ import print_function
+import sys
+import os
+import shutil
+import time
+import argparse
+
+VERSION = "0.1.0"
+
+
+class Error(Exception):
+    """Exception that indicates an error message."""
+
+    def __init__(self, message, code=1):
+        super(Error, self).__init__(message)
+        self.message = message
+        self.exit_code = code
+
+
+def delete_with_retry(folder):
+    """Try multiple times to delete a folder.
+
+    This is required on windows because of things like:
+    https://bugs.python.org/issue15496
+    https://blogs.msdn.microsoft.com/oldnewthing/20120907-00/?p=6663/
+    https://mail.python.org/pipermail/python-dev/2013-September/128350.html
+    """
+
+    for _i in range(0, 5):
```

Comment:
technically you can run this loop with just `for _ in range(0,5)`

Reply:
Good catch, fixed in `multipackage`, will get updated the next time we update the build scripts in this repository.